src/helper.js

```diff
@@ -20,6 +25,34 @@ Helper.config = require(path.resolve(path.join(
 	"config.js"
 )));
 
+function getVersion() {
+	var gitRef = getGitRef();
+	if (gitRef) {
+		return gitRef.replace(/^v/, "");
+	}
+	return pkg.version;
+}
+
+let _gitRef;
+function getGitRef() {
+	if (_gitRef) {
+		return _gitRef;
+	}
+	const commands = ["git describe HEAD", "git rev-parse --short HEAD"];
```

Comment:
It'll try all commands in the array, as soon as one succeeds it'll cache the result for subsequent invokations and then return the stdout. If the command fails it'll continue to the next one, and if all fails return `null`.